packages/fast-components-react-base/src/radio/radio.props.ts

```diff
@@ -0,0 +1,39 @@
+import * as React from "react";
+import { IManagedClasses, IRadioClassNameContract } from "@microsoft/fast-components-class-name-contracts-base";
+
+export enum RadioHTMLTags {
+    div = "div",
+    label = "label"
+}
+
+export interface IRadioHandledProps {
+    /**
+     * The HTML tag
+     */
+    tag?: RadioHTMLTags;
+
+    /**
+     * The checked state
+     */
+    checked?: boolean;
```

Comment:
@nalogiudice @nicholasrice @janechu:

Per our discussion on props, do we want "checked" for user facing? It maps to the component, but does it map to an intuitive API for radio? Is a radio in a group "checked" or is it "selected"? Per conversations on toggle.

Reply:
I think here "checked" is expected because it has a direct mapping to the HTML element - both input's and aria use "checked" for the "checkbox" type/role

Reply:
+1